warehouse/admin/views/sponsors.py

```diff
@@ -102,22 +103,17 @@ def sponsor_list(request):
     return {"sponsors": sponsors}
 
 
-def _handle_images(request, form):
+def _upload_image(image_name, request, form):
     sponsor_name = slugify(form.name.data)
-    if request.POST.get("white_logo") not in [None, b""]:
+    if request.POST.get(image_name) not in [None, b""]:
```

Comment:
(slight remark, but this is more simply expressed as `if request.POST.get(image_name):`, no ?